superset/assets/package.json

```diff
@@ -8,8 +8,9 @@
     "test": "spec"
   },
   "scripts": {
+    "start": "webpack",
```

Comment:
What is the intent of adding this script here? Are we using it in any stages?

Reply:
Here, we used jest with webpack to manage assets, styles, and compilation.

Reply:
Removed start:"webpack".